src/freenet/client/DefaultMIMETypes.java

```diff
@@ -65,14 +65,14 @@ protected static synchronized void addMIMEType(short number, String type) {
 	 */
 	protected static synchronized void addMIMEType(short number, String type, String[] extensions, String outExtension) {
 		addMIMEType(number, type);
-		Short t = Short.valueOf(number);
+		Short t = number;
```

Comment:
Looks like we don’t need `t` at all, do we? We can use `number` in every place that `t` is used.